jekyll/_cci2/audit-logs.adoc

```diff
@@ -63,4 +63,35 @@ The following are the system events that are logged. See `action` in the Field s
 - **version:** Version of the event schema. Currently the value will always be 1. Later versions may have different values to accommodate schema changes.
 - **scope:** If the target is owned by an Account in the CircleCI domain model, the account field should be filled in with the Account name and ID. This data is a JSON blob that will always contain `id` and `type` and will likely contain `name`.
 - **success:** A flag to indicate if the action was successful.
-- **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the unique ID assigned to this request by CircleCI).
\ No newline at end of file
+- **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the unique ID assigned to this request by CircleCI).
+
+== Request audit logs from the web app
```

Comment:
I wonder if "via" might work better than "from"? (same in the para below) ... Just because I read it like you're asking the web app for something, which you kind of are. I may have over thought. Feel free to ignore.

Reply:
Oh I thought we were trying to avoid using the word "via" but if that's not the case, then sure.